packages/adapter-node/README.md

```diff
@@ -40,11 +40,44 @@ Enables precompressing using gzip and brotli for assets and prerendered pages. I
 By default, the server will accept connections on `0.0.0.0` using port 3000. These can be customised with the `PORT` and `HOST` environment variables:
 
 ```
-HOST=127.0.0.1 PORT=4000 node build
+HOST=127.0.0.1 PORT=4000 node build/server
 ```
 
 You can specify different environment variables if necessary using the `env` option.
 
+## Middleware
+
+Adapter generates a [connect](https://github.com/senchalabs/connect) compatible middleware `(req, res, next) => {}`.
```

Comment:
My background isn't in Node. Does Express use Connect? I wonder if this could say `Express / Polka` instead

Reply:
changed.